readviz_prep.py

```diff
@@ -151,7 +157,4 @@ def main(args):
     )
     args = parser.parse_args()
 
-    if args.slack_channel:
-        try_slack(args.slack_channel, main, args)
-    else:
-        main(args)
+    main(args)
```

Comment:
The only thing is to remove the `--slack-channel` arg since the try_slack is gone. Otherwise LGTM.